alestojanovic/scavenger-hunt-v1
# fix: fixing issues with positioning and styling

### What did I do?
Fixed questionTooltipButton being too far up, form positioning and margins, container positioning, removed overriding CSS, switched to HashRouter to migrate deployment to gh-pages.

### How to test?
Run project locally and see how it looks or check it out on https://alestojanovic.github.io/scavenger-hunt-v1/.
